stdlib/bytes.mli

```diff
@@ -463,6 +463,106 @@ val of_seq : char Seq.t -> t
 (** Create a string from the generator
     @since 4.07 *)
 
+(** {6 Binary storage of integers} *)
```

Comment:
I would add a comment here that all these functions raise `Invalid_argument` if there's not enough space. Something like:
```
All following function raise [Invalid_argument] if the space needed at index [i] to decode 
or encode the integer is not available.
```